error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/TestNGToAssertJRules.java

```diff
@@ -72,6 +73,7 @@
 // XXX: As-is these rules do not result in a complete migration:
 // - Expressions containing comments are skipped due to a limitation of Refaster.
 // - Assertions inside lambda expressions are also skipped. Unclear why.
+@OnlineDocumentation
```

Comment:
Assuming these are migration rules, do we wish to keep these enabled by default in the future? Otherwise it might be nice not to add `@OnlineDocumentation` now where we can filter this one out when we start extracting these classes in docgen.

Reply:
Hmm, this one is likely to trigger if not explicitly disabled, so perhaps having the link would be nice? (I'm not too worried about breaking the URL later on, TBH.)